edge-middleware/feature-flag-split/components/vercel-icon.tsx

```diff
@@ -0,0 +1,40 @@
+const Vercel = ({ ...props }) => {
```

Comment:
It doesn't need a explicit type because of type inference?

Just in case its useful, you can type this using ComponentProps: `({ ...props }: React.ComponentProps<'svg'>)` to inherit all the types of a svg.

Reply:
This file is pure copy & paste since `@vercel/example-ui` doesn't work with App Router yet. Once it does, we can just delete this  file so I don't want to make any changes to it <picture data-single-emoji=":blob_smile:" title=":blob_smile:"><img class="emoji" width="20" height="auto" src="https://single-emoji.vercel.app/api/emoji/eyJhbGciOiJkaXIiLCJlbmMiOiJBMjU2R0NNIn0..WoeQj4tEavlJrnGp.d0wjEhT-BEqVCnJBMNxW0lFbfOiTmvIGRcrX2fgSrUsMdUy0oyddCV33qIqJI-zhEaSbymt8U2vPIAI27nyebLWj0B8afBuPaRRjIwbCuWGbVtrt3xikMK34zSk.hoqyQucRf4wlflgiYv_fTw" alt=":blob_smile:" align="absmiddle"></picture>